eng/Versions.props

```diff
@@ -247,6 +247,7 @@
     <IdentityServer4StoragePackageVersion>3.0.0</IdentityServer4StoragePackageVersion>
     <IdentityServer4EntityFrameworkStoragePackageVersion>3.0.0</IdentityServer4EntityFrameworkStoragePackageVersion>
     <MessagePackPackageVersion>2.1.90</MessagePackPackageVersion>
+    <MicrosoftAspNetCoreAuthPackageVersion>5.0.0-preview.8.20404.9</MicrosoftAspNetCoreAuthPackageVersion>
```

Comment:
Huh? We should not be explicitly setting this.

Reply:
Does it pack up the latest version if we don't set one?

Reply:
We should instead be using the automatically generated versions like we do for the rest of the packages built in this repo: https://github.com/dotnet/aspnetcore/blob/780d527f0b9df4594672c1a699eb2fc955de3d58/src/ProjectTemplates/Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in#L19

Reply:
And also this piece: https://github.com/dotnet/aspnetcore/blob/780d527f0b9df4594672c1a699eb2fc955de3d58/src/ProjectTemplates/Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj#L38